src/components/CondoCardModal.vue

```diff
@@ -74,14 +74,33 @@ export default {
       this.$store.commit('setActiveFeature', null);
     },
     addCondoRecords(state) {
-      if(this.$store.state.lastSearchMethod === "geocode") {
+      let mapUnitIds = function(id) {
         var i = 0;
```

Comment:
You can eliminate the need for this variable by using the second parameter of map (index) instead - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Cool! Thanks for the tip